docs/contributing.rst

```diff
@@ -1,22 +1,20 @@
 Contributing
 ============
 
-urllib3 is a community-maintained project and we happily accept contributions.
+Hip is a community-maintained project and we happily accept contributions.
 
 If you wish to add a new feature or fix a bug:
 
-#. `Check for open issues <https://github.com/urllib3/urllib3/issues>`_ or open
+#. `Check for open issues <https://github.com/python-trio/hip/issues>`_ or open
    a fresh issue to start a discussion around a feature idea or a bug. There is
    a *Contributor Friendly* tag for issues that should be ideal for people who
    are not very familiar with the codebase yet.
-#. Fork the `urllib3 repository on Github <https://github.com/urllib3/urllib3>`_
+#. Fork the `Hip repository on Github <https://github.com/python-trio/hip>`_
    to start making your changes.
 #. Write a test which shows that the bug was fixed or that the feature works
    as expected.
 #. Format your changes with black using command `$ nox -s blacken` and lint your
    changes using command `nox -s lint`.
-#. Send a pull request and bug the maintainer until it gets merged and published.
```

Comment:
Let's keep this line :)

Reply:
I tried pushing this myself but maybe you didn't check "Allow edit from maintainers"? https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
Done and done!